tools/adapters/su2/precice.hpp

```diff
@@ -0,0 +1,200 @@
+/*!
+* \file precice.hpp
+* \brief Header of adapter class for coupling SU2 with preCICE for FSI.
+* \author Alexander Rusch
+*/
+
+
+
+#include "/Users/alexanderrusch/GitHub/precice/src/precice/SolverInterface.hpp"
```

Comment:
This include is unlikely to work anywhere but on your system. Please use an include relative to the INCLUDE_PATH, which is precice/src